### DISCUSSION TITLE
Webpack `4.46.0` - query V5 - anyone come across issues.

### DISCUSSION THREAD

**Discussion by davidcarterschibsted** [TanStack/query#6317] [Q&A]
We have legacy migration project where we have both a backbone and react code running simultaneously, for this reason we are stuck on `webpack: 4.46.0` for a while. 

Has anyone managed to set up query V5 with this webpack version? Getting some errors over here?

**Comment by TkDodo** [comment]
we have an integration test with CRA4, which uses webpack 4, and it works fine:

https://github.com/TanStack/query/tree/e3a5974ba2d70b29ba3f45cba8c4f82d50407f2f/integrations/react-cra4

**Reply by davidcarterschibsted** [reply]
Thanks for the speedy response! 

Yes it works great, had some other webpack config issues which are resolved now.

Thanks again
